tests/test_pyu4v.py

```diff
@@ -1,2336 +1,2338 @@
-            mock_del.assert_called_once()
-
-    def test_get_slo_list(self):
-        """Test get_slo_list."""
-        slo_list = self.provisioning.get_slo_list()
-        self.assertEqual(self.data.slo_list['sloId'], slo_list)
-
-    def test_get_slo(self):
-        """Test get_slo."""
-        slo_details = self.provisioning.get_slo(self.data.slo)
-        self.assertEqual(self.data.slo_details, slo_details)
-
-    def test_modify_slo(self):
-        """Test modify_slo."""
-        with mock.patch.object(
-                self.provisioning, 'modify_resource') as mock_mod:
-            self.provisioning.modify_slo('Diamond', 'Diamante')
-            mock_mod.assert_called_once()
-
-    def test_get_srp(self):
-        """Test get_srp."""
-        srp_details = self.provisioning.get_srp(self.data.srp)
-        self.assertEqual(self.data.srp_details, srp_details)
-
-    def test_get_srp_list(self):
-        """Test get_srp_list."""
-        srp_list = self.provisioning.get_srp_list()
-        self.assertEqual(self.data.srp_list['srpId'], srp_list)
-
-    def test_get_compressibility_report(self):
-        """Test get_compressibility_report."""
-        report = self.provisioning.get_compressibility_report(self.data.srp)
-        self.assertEqual(
-            self.data.compr_report['storageGroupCompressibility'], report)
-
-    def test_is_compression_capable(self):
-        """Test is_compression_capable."""
-        compr_capable = self.provisioning.is_compression_capable()
-        self.assertTrue(compr_capable)
-
-    def test_get_storage_group(self):
-        """Test get_storage_group."""
-        sg_details = self.provisioning.get_storage_group(
-            self.data.storagegroup_name)
-        self.assertEqual(self.data.sg_details[0], sg_details)
-
-    def test_get_storage_group_list(self):
-        """Test get_storage_group_list."""
-        sg_list = self.provisioning.get_storage_group_list(
-            self.data.storagegroup_name)
-        self.assertEqual(self.data.sg_list['storageGroupId'], sg_list)
-
-    def test_get_mv_from_sg(self):
-        """Test get_mv_from_sg."""
-        mv_list = self.provisioning.get_mv_from_sg(
-            self.data.storagegroup_name_1)
-        self.assertEqual(self.data.sg_details[1]['maskingview'], mv_list)
-
-    def test_get_num_vols_in_sg(self):
-        """Test get_num_vols_in_sg."""
-        num_vols = self.provisioning.get_num_vols_in_sg(
-            self.data.storagegroup_name)
-        self.assertEqual(2, num_vols)
-
-    def test_is_child_sg_in_parent_sg(self):
-        """Test is_child_sg_in_parent_sg."""
-        is_child = self.provisioning.is_child_sg_in_parent_sg(
-            self.data.storagegroup_name_1, self.data.parent_sg)
-        self.assertTrue(is_child)
-        is_child2 = self.provisioning.is_child_sg_in_parent_sg(
-            self.data.storagegroup_name_2, self.data.parent_sg)
-        self.assertFalse(i
```

Comment:
Will review once line endings are fixed.

Reply:
File has been converted to unix